src/pages/blog/[...slug].js

```diff
@@ -30,18 +30,18 @@ export async function getStaticProps({ params }) {
 
 export async function getStaticPaths() {
   // getting all .md files from the blog directory, including subdirectories
-  const posts = glob.sync('data/blog/**/*.md');
+  const posts = glob.sync("src/data/blog/**/*.md");
   // converting the file paths to nested slugs (e.g., ['2023', '04', 'my-test-post'])
   const postSlugs = posts.map((file) => {
     // Split the file path and remove the first 'data/blog' part and '.md' extension
-    const pathParts = file.split('/').slice(2);
+    const pathParts = file.split("/").slice(2);
     // Remove the '.md' extension from the last part (post title)
     pathParts[pathParts.length - 1] = pathParts[pathParts.length - 1].replace(
       /\.md$/,
-      ''
+      "",
```

Comment:
Unnecessary trailing comma

Reply:
just a random formatter thing not sure why the old '' was there but prettier default converts `'` to `"` which is why that change occurred